docs/api/synthetics/monitors/update-monitor-api.asciidoc

```diff
@@ -0,0 +1,271 @@
+[[update-monitor-api]]
+== Update Monitor API
+++++
+<titleabbrev>Update monitor</titleabbrev>
+++++
+
+Updates a new monitor with the specified attributes. The required and default fields may vary based on the monitor type.
+
+=== {api-request-title}
+
+`PUT <kibana host>:<port>/api/synthetics/monitors/<config_id>`
+
+`PUT <kibana host>:<port>/s/<space_id>/api/synthetics/monitors/<config_id>`
+
+=== {api-prereq-title}
+
+You must have `all` privileges for the *Synthetics* feature in the *{observability}* section of the
+<<kibana-feature-privileges,{kib} feature privileges>>.
+
+
+[[update-monitor-api-path-params]]
+=== {api-path-parms-title}
+
+`config_id`::
+(Required, string) The ID of the monitor that you want to update.
+
+`space_id`::
+(Optional, string) An identifier for the space. If `space_id` is not provided in
+the URL, the default space is used.
+
+[[monitor-update-body]]
+==== Request Body
+
+The request body should contain the attributes of the monitor you want to update. The required and default fields differ depending on the monitor type:
+
+*Common Fields*:
+
+`name`:: (Required, string): The monitor's name.
```

Comment:
```suggestion
`name`:: (string): The monitor's name.
```

Not required.

Reply:
https://github.com/elastic/kibana/pull/169928/commits/81c6928c9bca943e24a338481562609f2025e029